src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -886,7 +899,8 @@ public Task<(long Etag, object Result)> WriteDatabaseRecordAsync(string database
             {
                 Name = databaseName,
                 RaftCommandIndex = index,
-                Record = record
+                Record = record,
+                DatabaseValues = databaseValues
```

Comment:
What is this database values that you have here?
If this is all the values for this db, we need to consider what happens if they are very large? Or if they have a large number of them?